app/models/item.rb

```diff
@@ -24,4 +25,8 @@ def show_status
       "retired"
     end
   end
+
+  def unique_categories
+    errors.add :base, "Item already belongs to that category" if categories.each{ |category| categories.include?(category)}
```

Comment:
Line is too long. [123/80]<br>Space missing to the left of {.<br>Space missing inside }.